package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "posthog-js",
-    "version": "1.123.0",
+    "version": "1.123.1",
```

Comment:
Wait - is this on purpose?

Reply:
Yeah, this commit has a tag attached to it, where I tried (and failed) to create a new release locally. Ideally should be merged.

Reply:
Ah wait we only have squash commits for this repo. Ah well it doesn't really matter - this PR is a minor bump wheras that was a patch

Reply:
have removed it